imports/api/server/methods.tests.js

```diff
@@ -0,0 +1,160 @@
+import { Meteor } from 'meteor/meteor';
+import { assert, expect } from 'meteor/practicalmeteor:chai';
+import { resetDatabase } from 'meteor/xolvio:cleaner';
+import { Factory } from 'meteor/dburles:factory';
+import { student, company } from '/imports/api/seed/users.js';
+import { job } from '/imports/api/seed/jobs.js';
+import { sinon } from 'meteor/practicalmeteor:sinon';
+import '/imports/api/server/methods.js';
+
+describe('Application methods', () => {
+  beforeEach(() => {
+    resetDatabase();
```

Comment:
This `beforeEach` will run on every test inside this same context described as **Application methods**

The function `resetDatabase()` as its name suggests resets the database, so here we are telling the tests to reset the database on every new test, it is a good practice to work with a clean DB on every single test